### ISSUE TITLE
[BUG] Errors randomly occur when pasting code into the editor. `v2.15.7 BETA`

### ISSUE THREAD

**Issue by F9y4ng** [violentmonkey/violentmonkey#1916]
First of all, thanks for fixing GM storage issue in v2.15.6 before I was about to report the ISSUE.

### Problem: 
When editing code locally, it will very randomly appear that an error occurs when pasting code into the editor, as follows:
#### ERROR
![PS_2023-10-04_15-19-23](https://github.com/violentmonkey/violentmonkey/assets/15855068/3d53c166-7663-4195-92d4-2c56f97d02f3)

#### CONSOLE
![PS_2023-10-04_15-21-03](https://github.com/violentmonkey/violentmonkey/assets/15855068/573675fe-cdde-42bc-ab6d-3fcfe83e6cc0)

I don't know how to reproduce it 100%, and although it doesn't affect normal operation (refreshing will restore it), the error message that appears at some point always interrupts editing.

#### Environment:
- OS: Windows
- Browser: Chrome 117 
- Violentmonkey Version: v2.15.7 BETA

**Comment by gera2ld** [comment]
https://github.com/violentmonkey/violentmonkey/blob/8d6f3264a5d8c56784036b858718a0df5fec7b35/src/options/views/edit/values.vue#L333

Seems `cur.jsonValue` could be empty here.

**Comment by tophf** [comment]
Should be fixed in 5b6e5596, could you please verify it?

**Comment by tophf** [comment]
No, it's not, but I found how to reproduce it reliably, investigating why jsonValue is null...

**Comment by F9y4ng** [comment]
> Until the new version is released, try not to switch to the values tab unnecessarily :-)

OK, thanks! 
I can also reproduce it 100% by the way you said, I will avoid it.
